pyanaconda/install.py

```diff
@@ -131,6 +133,17 @@ def doConfiguration(storage, payload, ksdata, instClass):
     # kickstart file over if one exists).
     _writeKS(ksdata)
 
+    if willSnapShotRoot:
+        with progress_report(_("Configuring root snapshot")):
+            blivet.autopart.setUpRootSnapShot(storage)
+
+        with progress_report(_("Generating initramfs")):
+            payload.recreateInitrds()
+
+        if not ksdata.bootloader.disabled and ksdata.bootloader != "none":
+            with progress_report(_("Installing bootloader")):
+                writeBootLoader(storage, payload, instClass, ksdata)
```

Comment:
This should be without a separate progress report step (just like in case of the liveCD+BtrfsBoot combination above) unless you want to move the check somewhere above and reflect it in the total number of steps.